clang/test/CodeGen/profile-filter.c

```diff
@@ -4,7 +4,8 @@
 // RUN: %clang_cc1 -fprofile-instrument=clang -fcoverage-mapping -dump-coverage-mapping -fprofile-list=%t-func.list -emit-llvm %s -o - | FileCheck %s --check-prefix=FUNC
 
 // RUN: echo "src:%s" | sed -e 's/\\/\\\\/g' > %t-file.list
-// RUN: %clang_cc1 -fprofile-instrument=clang -fcoverage-mapping -dump-coverage-mapping -fprofile-list=%t-file.list -emit-llvm %s -o - | FileCheck %s --check-prefix=FILE
+// RUN: cd %S
+// RUN: %clang_cc1 -fprofile-instrument=clang -fcoverage-mapping -dump-coverage-mapping -fprofile-list=%t-file.list -emit-llvm profile-filter.c -o - | FileCheck %s --check-prefix=FILE
```

Comment:
Is `%s` an absolute or relative path? Either way, this only tests one of the cases. We should test both. I.e. if `%s` is the absolute path, there should also be a test for `"src:profile-filter.c"`. However, it `%s` is the absolute path, wasn't this already working before this patch?

Reply:
Yes, I misunderstood the concept wrt ninja build and fun.list. Only modification is required in fun.list to supply full path, not required to update ProfileList.cpp

 Apologies for taking so much time to realize this simple concept.

Reply:
Shall I close this PR or it continue in case it will be useful for someone else?